tox.ini

```diff
@@ -27,6 +29,7 @@ deps =
     django20: Django>=2.0,<2.1
     django21: Django>=2.1,<2.2
     django22: Django>=2.2a1,<3.0
+    django30: Django>=3.0,<3.1
     six
```

Comment:
I don't think six needs to be listed here now it's a dependency of easy-thumbnails